app/controllers/application_controller.rb

```diff
@@ -2,10 +2,19 @@ class ApplicationController < ActionController::Base
   before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  helper_method :friends_and_myself
+
   private
 
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: %i[name lastname])
     devise_parameter_sanitizer.permit(:account_update, keys: %i[name lastname])
   end
+
+  def friends_and_myself(user)
```

Comment:
- [ ] **[OPTIONAL]** `friends_and_myself` method doesn't really belong in here. It considered as business logic (CRUD operations), and you want to always wrap business logic in your models.